jekyll/_cci2/monitoring.adoc

```diff
@@ -14,10 +14,14 @@ toc::[]
 
 To enable system and Docker metrics forwarding to either AWS Cloudwatch or Datadog, navigate to your CircleCI Management Console, select Settings from the menu bar and scroll down to enable the provider of your choice (your-circleci-hostname.com:8800/settings#cloudwatch_metrics).
 
+// Can this just say 'metrics' rather than system and Docker metrics?
```

Comment:
I think that's fair, no need to over complicate it.

Reply:
Done